drivers/source/SFDP.cpp

```diff
@@ -338,11 +415,49 @@ int sfdp_iterate_next_largest_erase_type(uint8_t &bitfield,
         type_mask = type_mask >> 1;
     }
 
-    if (i_ind == 4) {
+    if (idx == -1) {
         tr_error("No erase type was found for current region addr");
     }
     return largest_erase_type;
 }
 
+int sfdp_detect_device_density(uint8_t *bptbl_ptr, sfdp_bptbl_info &bptbl_info)
```

Comment:
Is this also another publicly available interface function ?

Reply:
Yes, this time that is the case.